src/verso/Verso/Doc/Elab/Monad.lean

```diff
@@ -344,10 +182,13 @@ instance : MonadWithOptions PartElabM := inferInstanceAs <| MonadWithOptions (Re
 def PartElabM.withFileMap (fileMap : FileMap) (act : PartElabM α) : PartElabM α :=
   fun ρ ρ' σ ctxt σ' mctxt rw cctxt => act ρ ρ' σ ctxt σ' mctxt rw {cctxt with fileMap := fileMap}
 
+/--
+Text elaboration monad. Can modify the DocElabM.State, but can only read from the PartElabM.state
```

Comment:
```suggestion
Text elaboration monad.

This monad can produce content, but it can't modify the structure of the surrounding document. It can
observe this structure, however.

This means  it can modify the `DocElabM.State`, but it can only read from the `PartElabM.State`.
```